index.html

```diff
@@ -40,12 +35,12 @@
       </div>
     </section>
 
-    <section class="content">
-      <p class="boldtext">
+    <div class="content">
+      <h1 class="boldtext">
         I’m Megha<br />
```

Comment:
[OPTIONAL]

- [ ] Please change this div to a section instead as it is the main section for your content.